src/components/MenuItem.tsx

```diff
@@ -15,7 +15,7 @@ import getButtonState from '@libs/getButtonState';
 import Parser from '@libs/Parser';
 import type {AvatarSource} from '@libs/UserUtils';
 import variables from '@styles/variables';
-import {checkIfActionIsAllowed} from '@userActions/Session';
+import {callFnIfActionIsAllowed} from '@userActions/Session';
```

Comment:
Ideally I think this would be `callFunctionIfActionIsAllowed` instead of using the shortened `fn`